synapse/lib/stormlib/infosec.py

```diff
@@ -594,7 +594,7 @@ class CvssLib(s_stormtypes.Lib):
                                 environmental score, overall score, and normalized vector string.
                                 The normalized vector string will have metrics ordered in
                                 specification order and metrics with undefined values will be
-                                removed. Example:
+                                removed. Example::
```

Comment:
Extra colon?

Reply:
This creates an RST literal block for doc rendering